# message to store distance and dimensions of an object.

MinMaxMean x
MinMaxMean y
MinMaxMean z

MinMaxMean distance

uint8 classNr
